spring-cloud-dataflow-rest-client/src/main/java/org/springframework/cloud/dataflow/rest/client/TaskOperations.java

```diff
@@ -121,30 +121,16 @@ public interface TaskOperations {
 	 */
 	TaskExecutionResource taskExecutionStatus(long id);
 
-	/**
-	 * Return the task execution log.  The platform from which to retrieve the log will be set to {@code default}.
-	 *
-	 * @param externalExecutionId the external execution identifier of the task execution.
-	 * @return {@link String} containing the log.
-	 */
-	String taskExecutionLog(String externalExecutionId);
```

Comment:
This would be a breaking change (on the Java interface API anyways). Maybe we should leave these in place, delegate to the new one, and mark them as `@deprecated`. Wdyt?

Reply:
👍